core/node/storage/migrations/000006_fixed_stream_partitions.up.sql

```diff
@@ -0,0 +1,40 @@
+DO $$
+	DECLARE
+
+	suffix CHAR(2);
+	i INT;
+
+	BEGIN
+
+	FOR i IN 0..255 LOOP
+		suffix = LPAD(TO_HEX(i), 2, '0');
+
+		EXECUTE 'CREATE TABLE IF NOT EXISTS miniblocks_' || suffix || ' (
+  			stream_id CHAR(64) NOT NULL,
+  			seq_num BIGINT NOT NULL,
+  			blockdata BYTEA NOT NULL,
```

Comment:
bunch of rows are updated with storage qualifies now, please take a look into update 0005 migration.